skglm/tests/test_prox_newton.py

```diff
@@ -20,14 +20,15 @@ def test_alpha_max(X_density):
 
     log_datafit = compiled_clone(Logistic())
     l1_penalty = compiled_clone(L1(alpha_max))
-    w = ProxNewton().solve(X, y, log_datafit, l1_penalty)[0]
+    w = ProxNewton(fit_intercept=False).solve(X, y, log_datafit, l1_penalty)[0]
 
     np.testing.assert_equal(w, 0)
 
 
-@pytest.mark.parametrize("rho, X_density", product([1e-1, 1e-2], [1., 0.5]))
-def test_pn_vs_sklearn(rho, X_density):
-    n_samples, n_features = 11, 19
+@pytest.mark.parametrize("rho, X_density, fit_intercept",
+                         product([1e-1, 1e-2], [1., 0.5], [True, False]))
```

Comment:
@Badr-MOUFAD is there a particular thing to test rho=1 ? For me if it works for rho=0.1 it works for rho=1. We don't test anything particular for rho=1.

If not, it can be removed (the test will be simpler and have the same coverage)

Reply:
I have no strong opinion.
It's just a sanity check that we've used to implement.

Reply:
Then I have a strong opinion :) 
It's a sanity check when we use benchopt, to check that the objective doesn't move. Here, we don't check that, so this parameter can be removed to make the test simpler. Just use a hardcoded rho=0.1